server/middleware/render.js

```diff
@@ -1,29 +1,32 @@
+/*
+ enable babel server env plugin transform
+ should be placed on top-level before importing any js, jsx
+ */
+process.env.BABEL_ENV = 'server';
+
 import React from 'react';
```

Comment:
[eslint] Import in body of module; reorder to top. (import/first). should we remove this rule

Reply:
i think we could keep this eslint rule,  this rule is almost used for client js. In server side, some files need to be parsed by `babel` maybe some env variable is needed.

Reply:
so you need to add a ``/* eslint-disable import/first */`` on the top in some files

Reply:
all right